### DISCUSSION TITLE
How do I execute a program in subfolder?

### DISCUSSION THREAD

**Discussion by AMDphreak** [cmderdev/cmder#2501] [Q&A]
Using `./folder/file.exe` does not work, because Cmder doesn't recognize `.` (the current directory)

**Comment by daxgames** [comment]
This is not unix/linux bash.  `folder\file.exe`

**Reply by AMDphreak** [reply]
I am aware of that. Which is why I specifically said "because Cmder doesn't recognize `.` (the current directory." I did in fact do the obvious thing and try executing the path `.\folder\file.exe`. I also tried `folder\file.exe` which did not work. EDIT: I may have tried every combination including `folder/file.exe` except `folder\file.exe`. Also Cmder's behavior is inconsistent. It is legal to run `ls ./folder` and `ls`'s output is all in unix `/` notation. Inconsistent philosophy is a problem. Also it is important to note that every other shell that either runs in Windows or emulates Linux environment supports `.` when present in its native syntax such as in PowerShell: `dir .\folder` is valid in PowerShell, but this same syntax cannot be used outside the scope of the `dir` command in Cmder. The default syntax in Cmder should be domain-agnostic, so I should be able to execute using Linux or Windows syntax.

**Comment by 296951** [comment]
Sorry your 'specific' statment was not so obvious to me.

What you are complainign about works fine for me:

![image](https://user-images.githubusercontent.com/76230985/112554711-2c724400-8d9d-11eb-937e-d2396dc3b445.png)
